schedule-additional-tests

```diff
@@ -26,5 +26,15 @@ for x in `grep ADDITIONAL_TESTS=  $CONFIG_MAP | grep "SCRAM_ARCH=$ARCHITECTURE;"
     echo "EVENTS=10" >> ${nfile}
     rm -f ${pfile}
   fi
+  if [ "$x" = "profiling" ] ; then
+    PROFILING_WORKFLOWS=$(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||' | tr ',' ' ')
+    for wf in $PROFILING_WORKFLOWS; do
+      nfile=$WORKSPACE/auto-$x-${wf}-properties
+      cp ${pfile} ${nfile}
+      echo "PROFILING_WORKFLOWS=$PROFILING_WORKFLOWS" >> ${pfile}
```

Comment:
you should use `${wf}` instead of `$PROFILING_WORKFLOWS` here. Also use `${nfile}` . You are also missing 
```
echo "PROFILE=pp" >> ${nfile}
```

Reply:
I changed the parameters of the job to not use PROFILE or WORKFLOWS.